PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,6 +12,11 @@
 - remove
 - modify
 
-## QA steps (optional)
+## Reason for Change
+- #TICKET_NUMBER
+- If the reason for this PR's code changes are not clear in the issue, state value/impact
+
+## QA steps
+- Either list QA steps or reasoning you feel QA is unnecessary
```

Comment:
Unless we make the QA process more formalized, I don't think this is particularly useful. Who will do the QA besides the engineer who submits the PR?

Reply:
I do sometimes depending on how big the changes are. I also only fill out this section when QA requires manual steps and it outside of simply running unit tests. The only time manual steps are usually needed is when it's db migration, or code that is outside of the application like admin scripts.

Reply:
I think this will most commonly be filled out with run the tests.

Reply:
In that case, I feel like it's generally useful to coordinate with the oncall engineer so that the deployment goes smoothly. Since the oncall engineer isn't responsible for reviewing all PRs, filling the QA step here doesn't replace that.

Reply:
@ebezzi what do you mean by more formalized? 

This section is meant to indicate - how did you as the engineer test your code? How do you know its working? How would you test your code? And if its not something that requires testing, indicate why.

Reply:
I think we might use a better term than QA, since I think we're conflating this with PM/stakeholder QA testing.  Maybe just call this section "Testing".  I see this section as explaining to the reviewer how you made sure the change is ready to go.

  I would use this to call out things like "Unit tested only", "tested in rdev, verifying feature worked by... ", "manually ran pipeline locally with these results: ..."

Reply:
@atolopko-czi @ebezzi @Bento007 thanks for the feedback here. I have updated the template, please let me know what you think